corehq/apps/accounting/invoicing.py

```diff
@@ -368,8 +368,6 @@ def _generate_customer_invoice(self):
             date_start=self.date_start,
             date_end=self.date_end
         )
-        if not is_new_invoice:
-            raise InvoiceAlreadyCreatedError("invoice id: {id}".format(id=invoice.id))
```

Comment:
Are you going to add this back after you run the fill_subscriptions command?

Reply:
Hey, I was planning to make it a limited released just to run this command. Therefore, It wont be a part of master code.